.travis.yml

```diff
@@ -6,11 +6,6 @@ services:
   - docker
 
 env:
-  global:
```

Comment:
why did you remove the env vars?

Reply:
Because the variables in `.travis.yaml` override the variables set in my Travis build. The commit message has more info. Idk how to link a commit, but its commit `a3e55cb`.